### PR TITLE
Invert Profile Relationships CU-2jabv0z

### PR DESCRIPTION
**Pull request by bwpikaard**

### CODE REVIEW

**Review by gankoji — APPROVED** [review]

**Comment by 0c370t** [comment]
Task linked: [CU-2jabv0z Invert Profile Ownership](https://app.clickup.com/t/2jabv0z)

**Comment by bwpikaard** [comment]
Do not merge. Need prep/reset/migration for the database.

**Comment by zachstence** [comment]
Converted to draft until it's ready to merge

**Pull request merged**
